### DISCUSSION TITLE
Start repo search and filer

### DISCUSSION THREAD

**Discussion by saschamcdonald** [community/community#38591] [Other Feature Feedback, Questions, & Ideas]
I have some issues searching my started repo's. Please let me know and also sorry if I'm just missing functionality that already exists: 

Feature request to access started repos:

Stars:
iOS app and website:
Filers and Order By would be great

An option to share stars in an Organisation and also with externally members would be awesome

Thanks again,

Sascha

**Comment by david-engelmann** [comment]
The stars section should have the same search capabilities as the repo section (see screenshot below)!

![image](https://user-images.githubusercontent.com/81196676/200639092-1ca41287-f6dd-490e-9979-8bd9e1fa8a0a.png)

**Comment by ossanna16** [comment]
Hi there @saschamcdonald and welcome to our community! Thank you for the feature request!

**Reply by Santiagosaiz** [reply]
@Santiagosaiz4#inittowinit
